bazel-java-deps.bzl

```diff
@@ -20,6 +21,7 @@ def install_java_deps():
             "com.github.ghik:silencer-lib_2.12.11:1.6.0",
             "com.github.ghik:silencer-plugin_2.12.11:1.6.0",
             "com.github.pureconfig:pureconfig_2.12:0.8.0",
+            maven.artifact("com.github.pureconfig", "pureconfig-macros_2.12", "0.8.0", neverlink = True),
```

Comment:
this is a global config, right?
are you sure the other components do not need it in run-time?

Reply:
Only going by CI here. It is only used by bindings-akka afaik so if the JSON API doesn’t need it, I expect other clients probably also don’t need it.

Reply:
sounds good.... I forgot that we test `daml sandbox` startup as part of the CI build. So, we are good.